<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Translater</class>
 <widget class="QWidget" name="Translater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string> 极简翻译</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/ya.ico</normaloff>:/icons/ya.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="Icon">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
border-image: url(:/icons/ya.ico);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Title">
          <property name="styleSheet">
           <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	border: None;
}</string>
          </property>
          <property name="text">
           <string>极简翻译</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="TextFrom">
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">QPlainTextEdit {
    background-color: rgba(8, 8, 8, 150);
    color: #ffff00;
        border: 1px solid #ff00ff
}
</string>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="TextTo">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">QPlainTextEdit {
    background-color: rgba(8, 8, 8, 150);
    color: #00ffff;
    border: 1px solid #ff00ff
}
</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pBtnCopyTranlate">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="toolTip">
           <string>复制翻译结果</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/drip_copy.ico</normaloff>:/icons/drip_copy.ico</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bBtnClean">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string>清空文本</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/drip_trash.ico</normaloff>:/icons/drip_trash.ico</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pBtnEnToZh">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string>翻译为中文</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/empty_zh.ico</normaloff>:/icons/empty_zh.ico</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pBtnZhToEn">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string>翻译为英文</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/black_en.ico</normaloff>:/icons/black_en.ico</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pBtnPin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>30</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string>防止隐藏</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/drip_pin.ico</normaloff>:/icons/drip_pin.ico</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
